ros2_generate_interface_docs/ros2_generate_interface_docs/templates/msg.html.em

```diff
@@ -16,27 +16,27 @@ import time
       <h2>File: <span class="filename">@(interface_package)/@(interface_name).@(ext)</span></h2>
       <h2>Raw Message Definition</h2>
       <div class="raw-msg">
-      @[for line in raw_text.splitlines()]@
-        @{text = line.strip('#')}@
-        @[if '#' in line]
+@[for line in raw_text.splitlines()]@
+@{  text = line.strip('#')}@
+@[  if '#' in line]
```

Comment:
This needs a trailing `@` to suppress the newline from ending up in the generated code.

Same below.